opm/simulators/wells/MultisegmentWellEval.cpp

```diff
@@ -338,6 +320,39 @@ assembleICDPressureEq(const int seg,
                            linSys_,
                            FluidSystem::phaseIsActive(FluidSystem::waterPhaseIdx),
                            FluidSystem::phaseIsActive(FluidSystem::gasPhaseIdx));
+
+    assembleAccelerationAndHydroPressureLosses(seg, well_state, use_average_density);
+}
+
+template<typename FluidSystem, typename Indices, typename Scalar>
+void
+MultisegmentWellEval<FluidSystem,Indices,Scalar>::
+assembleAccelerationAndHydroPressureLosses(const int seg,
+                                           WellState& well_state,
+                                           const bool use_average_density)
+{
+    auto& ws = well_state.well(baseif_.indexOfWell());
+    auto& segments = ws.segments;
```

Comment:
nitpicks here,  `segments` is used in the later stage. The above two lines can be moved to line 342.